src/canvas/static_canvas.class.ts

```diff
@@ -722,6 +716,10 @@ export class StaticCanvas<
     }
   }
 
+  drawControls(ctx: CanvasRenderingContext2D) {
+    // Static canvas has no controls
+  }
```

Comment:
shouldn't we alter `renderCanvas`?
Or is too much?

Reply:
I wanted to do that, but since it does it in 2 parts is not as simple as running an extra effect before or after the super call.